canvas_api_client/README.md

```diff
@@ -33,20 +35,38 @@ Contributing
 
 This project is tested with python3, and additionally has mypy integration.
 
+Note: before building, make sure to bump the `__version__` in the `setup.py` file.
+
+#### Building Wheels ####
+
 Building the wheel:
 
     pip install -r requirements.txt
```

Comment:
do we need this step? i was just thinking about this for something i was working on today. as long as we have setuptools and wheel available, which i think `virtualenv` and `venv` ship those by default, we should be able to just run the `bdist_wheel` command, right? b.c. we specify our dependencies in `setup.py`.. not a huge thing. 

the only reason i'm commenting on it b.c. i was thinking about this today

Reply:
You mean without the pip install? Probably right.

Reply:
Yep, tried it out and that is right. Let me remove this.

Reply:
+1